mlf_core/create/templates/common_mlflow_files/{{ cookiecutter.common_mlflow }}/README.rst

```diff
@@ -16,18 +16,13 @@
         :alt: mlf-core lint
 
 {% if is_open_source %}
-.. image:: https://readthedocs.org/projects/{{ cookiecutter.project_name }}/badge/?version=latest
-        :target: https://{{ cookiecutter.project_name }}.readthedocs.io/en/latest/?badge=latest
+.. image:: https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}/actions/workflows/publish_docs.yml/badge.svg?branch=main
```

Comment:
hardcoding the main branch may to always work.

Some people are using master etc.
Is there a better solution?

Reply:
Good point I'll look into it

Reply:
Not sure what to do honestly. I hate the change by Github. Either we link to just publish_docs.yml or we hardcode the development branch?